packages/svrx/lib/config-list.js

```diff
@@ -17,6 +17,12 @@ module.exports = {
     group: GROUPS.CORE,
     ui: false,
   },
+  path: {
```

Comment:
Maybe `svrxPath` are better than `path`？ `path` are too versatile

Reply:
`path` here has a same meaning like `path` in plugin configs, maybe the consistancy should be kept.